### PR TITLE
Add Vaka

### PR DESCRIPTION
**Pull request by knatten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ A curated list of awesome [Fuse](https://www.fusetools.com/) libraries, tools, a
 - [GameCalculator](https://itunes.apple.com/us/app/gamecalculator/id952709405) - A custom game calculator for parties [(​_iOS_​)](https://itunes.apple.com/us/app/gamecalculator/id952709405) [(​_Android_​)](https://play.google.com/store/apps/details?id=com.GameCalculator).
 - [Timecircle](https://itunes.apple.com/bt/app/timecircle/id1068220814) - Simple yet effective and good-looking stop watch [(​_iOS_​)](https://itunes.apple.com/bt/app/timecircle/id1068220814) [(​_Android_​)](https://play.google.com/store/apps/details?id=com.vegardstrand.TimeCircle).
 - [Timestealer](https://itunes.apple.com/us/app/timestealer/id1073144825) - Track how much your meetings are costing you and your company [(​_iOS_​)](https://itunes.apple.com/us/app/timestealer/id1073144825) [(​_Android_​)](https://play.google.com/store/apps/details?id=com.Timestealer).
+- [Vaka](https://itunes.apple.com/us/app/vaka/id1077345742) - Vaka is a game for focused and stable mind. [(​_iOS_​)](https://itunes.apple.com/us/app/vaka/id1077345742) [(​_Android_​)](https://play.google.com/store/apps/details?id=com.Vaka).
```
Comment (vinkla) [inline README.md]: Please remove the dot at the end of `Vaka is a game for focused and stable mind.`

**Comment by vinkla** [comment]
Thanks for your contribution! I've just made a little comment you could take a look at before I merge this.

**Comment by vinkla** [comment]
Thanks!

**Comment by knatten** [comment]
Sorry about the extra period, I misunderstood the point point about "Start the description with a capital and end with a full stop/period." in `CONTRIBUTING.md`

**Comment by vinkla** [comment]
Yeah, sorry about that. Maybe I should make it more clear.

**Pull request merged**